tests/main/views/test_marketplace.py

```diff
@@ -451,7 +451,7 @@ def test_dos_brief_has_application_stats_correctly_when_no_applications(self):
 
         assert started_responses_section.xpath('div[@class="big-statistic"]/text()')[0] == '0'
         assert completed_responses_section.xpath('div[@class="big-statistic"]/text()')[0] == '0'
-        assert started_responses_section.xpath('div[@class="statistic-name"]/text()')[0] == "Started applications"
+        assert started_responses_section.xpath('div[@class="statistic-name"]/text()')[0] == "Incomplete applications"
```

Comment:
Not sure I understand it as serializing, but you've had to update the same assertion three times, and those repeated things could be in a function. (Assuming we really need to assert the same thing in three different tests.)